gulpfile.js

```diff
@@ -17,8 +17,13 @@ var browserify = require('browserify');
 var source = require('vinyl-source-stream');
 var merge = require('merge-stream');
 var collapse = require('bundle-collapser/plugin');
-var argv  = require('yargs').argv
+var argv  = require('yargs')
```

Comment:
Though, since this one will grow, I would keep it outside the other dependencies for a better readability of dependencies :)

Maybe:

```javascript
var yargs= require('yargs');

// ...

var argv = yargs
  .option('force-output', {default: 'false'})
  // ..
  .argv;
```

Reply:
or like that: https://github.com/chartjs/chartjs-plugin-datalabels/blob/master/gulpfile.js#L20

Reply:
but I think I prefer the first solution: `var yargs= require('yargs');`

Reply:
Yes, I think it makes more sense to keep all the dependencies small and together.